docs/design.md

```diff
@@ -3,76 +3,97 @@ Kristen Mason & Hilary Nelson
 
 #### Design Decisions & Alternative Designs Considered
 
-We tried to make the design as similar as possible between the new task list view and the task view to provide the user with consistency throughout the app. However, we were careful to distinguish the pages enough so as to avoid confusion between the two (the task list view or the task view). We did this by keeping the same styling across both, and leaving the existing style for editing the item’s name, but changing the title at the top of the page and the coloring of tasks vs. task lists. We also decided that deleting a task list is more significant than deleting a task, and thus you can only delete one task list at a time, with the trash icon right on the right side of the task list. We also updated our view option to indicate clearly which option is currently selected by updating the option to have inverse coloring.
+Because our web app has from the start been designed with a mobile layout in mind, we found it very straightforward to preserve our previous design decisions from labs 1-5 while working on Lab 6. The Moto G4 layout was very similar to the simulated Apple device layouts in terms of dimensions, so we found ourselves really only having to make superficial changes to the CSS in order to make all of our task item elements appear on the screen as they had in our web app. Other than that, our React Native version of the app appears nearly identical to the version of the app we published for Lab 5– which was our goal.
```

Comment:
It looks as though you used Cordova, which provides a native container for a web app, and not React Native, which uses the react framework to create a native application but which doesn't use CSS or any HTML components.